### PR TITLE
Cypress/E2E: Add filter for tests not for cloud

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add filter for tests not for cloud. Those tests typically requires server configuration that are not writable in Cloud edition.

### CODE REVIEW

File: e2e/cypress/integration/system_console/site_configuration/link_customization_spec.js [hunk 1]
```diff
@@ -17,17 +18,15 @@ describe('SupportSettings', () => {
...
 
     const defaultTosLink = 'https://about.mattermost.com/default-terms/';
-    const defaultPrivacyLink = 'https://about.mattermost.com/default-privacy-policy/';
 
     let testTeam;
 
     beforeEach(() => {
-        // # as many of the tests logout the user, ensure it's logged
-        // in as an admin before each test
+        // # Login as admin and reset config
```
Comment (josephbaylon) [inline e2e/cypress/integration/system_console/site_configuration/link_customization_spec.js:27]:
```suggestion
        cy.shouldNotRunOnCloudEdition();

        // # Login as admin and reset config
```

**Review by josephbaylon — APPROVED** [review]
LGTM. Just one missed line below I think.

File: e2e/cypress/integration/system_console/site_configuration/link_customization_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/system_console/site_configuration/link_customization_spec.js:27]: Thanks and good catch! Actually, I've already moved the tests that were not for cloud to `link_customization_not_cloud_spec`. I just missed to remove the corresponding `@not_cloud` group tag here. Now fixed.

**Comment by saturninoabril** [comment]
@josephbaylon I've pushed another commit that moved MM-T1775 and MM-T1776 into its own spec file since they are not for cloud since `ServiceSettings.MaximumLoginAttempts` are not writable. You might want to take a look again.

**Review by josephbaylon — APPROVED** [review]
LGTM. Retested updated changes. Passing locally.

**Review by furqanmlk — APPROVED** [review]
Thanks @saturninoabril

**Review by prapti — APPROVED** [review]
LGTM! Thank you Saturn!

**Pull request merged**